### ISSUE TITLE
Logs should be grouped when not attached to a terminal or using plain console

### ISSUE THREAD

**Issue by eriwen** [gradle/gradle#2221]
### Expected Behavior
Group logs as is done when attached to a terminal. Same as #1949 

### Current Behavior
Currently, logs in CI are still ungrouped. This is especially useful as we introduce more parallelism

### Context
Follow on from #1949 to complete this for 4.1. This was taken out of 4.0 due to the risks of holding JVMs open as caught by `ToolingApiIntegrationTest`

Related to #2240 maybe

**Comment by eriwen** [comment]
I'm afraid we're going to have to work through https://github.com/gradle/gradle/issues/2240 before we can achieve this. Calling it blocked for now.

**Comment by robbie-demuth** [comment]
Are there any updates on this? This issue mentions that #2240 need to be completed first, but that issue was closed as a duplicate of this issue. Grouping parallel output would make debugging CI builds orders of magnitude easier

**Comment by eriwen** [comment]
Perhaps you want to pick this up @ghale?

**Comment by adammurdoch** [comment]
This is implemented now. The plain console now works exactly the same way as the verbose console, minus the text decoration and the 'work in progress' section at the bottom of the screen.
